src/Symfony/Component/Routing/Tests/Matcher/ExpressionLanguageTest.php

```diff
@@ -0,0 +1,84 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\Routing\Tests\Matcher;
+
+use PHPUnit\Framework\TestCase;
+use Symfony\Component\ExpressionLanguage\ExpressionLanguage;
+use Symfony\Component\Routing\Matcher\ExpressionLanguageProvider;
+use Symfony\Component\Routing\RequestContext;
+
+class ExpressionLanguageTest extends TestCase
+{
+    private $context;
+    private $expressionLanguage;
+
+    public function setUp(): void
```

Comment:
this is meant to be protected from parent afaik

Reply:
Done